# HG changeset patch
# User Rik <rik@octave.org>
# Date 1409370291 25200
#      Fri Aug 29 20:44:51 2014 -0700
# Node ID 1c4994e6696219b09925700834451aa028d0a699
# Parent  edf5d63c82e1fc837c2bad5d748c6be3a33beae8
doc: Remove references to deprecated bicubic function in manual.

* interp.txi: Remove bicubic DOCSTRING reference.

diff --git a/doc/interpreter/interp.txi b/doc/interpreter/interp.txi
--- a/doc/interpreter/interp.txi
+++ b/doc/interpreter/interp.txi
@@ -171,12 +171,8 @@ respectively.
 The result of this code can be seen in @ref{fig:interpn}.
 
 @float Figure,fig:interpn
 @center @image{interpn,4in}
 @caption{Demonstration of the use of @code{interpn}}
 @end float
 @end ifnotinfo
 
-In additional the support function @code{bicubic} that underlies the
-cubic interpolation of @code{interp2} function can be called directly.
-
-@DOCSTRING(bicubic)
